scripts/splunkscript/readme.md

```diff
@@ -0,0 +1,68 @@
+﻿# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
as this is "md" file, the hash characters here would result in setting the copyright lines as HEADING1.

I would suggest to remove the copyright from readme, as is done elsewhere in the repo.